server/services/zigbee2mqtt/utils/convertValue.js

```diff
@@ -14,11 +14,26 @@ function convertValue(feature, state) {
       result = state === 'ON' ? 1 : 0;
       break;
     }
-    default:
-      if (typeof state === 'string') {
-        throw new Error(`Zigbee2mqqt don't handle value "${state}" for feature "${feature}".`);
-      }
+    // Case for Button devices
+    case 'click': {
       result = state;
+      break;
+    }
+
+    default: {
+      switch (typeof state) {
+        case 'boolean': {
+          result = state ? 1 : 0;
+          break;
+        }
+        case 'number': {
+          result = state;
+          break;
+        }
+        default: // string
+          throw new Error(`Zigbee2mqqt don't handle value "${state}" for feature "${feature}".`);
+      }
+    }
```

Comment:
Checking formatting...
services/zigbee2mqtt/utils/convertValue.js
Code style issues found in the above file(s). Forgot to run Prettier?

https://circleci.com/gh/atrovato/Gladys/348#tests/containers/0

Reply:
Yes, it was late yesterday and I forgot to run prettier after my last modification...